src/main/scala/mesosphere/marathon/core/appinfo/impl/DefaultInfoService.scala

```diff
@@ -83,7 +83,7 @@ private[appinfo] class DefaultInfoService(
             None
         def apps: Option[Seq[AppInfo]] =
           if (groupEmbed(GroupInfo.Embed.Apps))
-            Some(ref.apps.toIndexedSeq.flatMap(a => infoById.get(a.id)).sortBy(_.app.id))
+            Some(ref.apps.values.toIndexedSeq.flatMap(a => infoById.get(a.id)).sortBy(_.app.id))
```

Comment:
Looks like `ref.apps.keys.flatMap(infoById.get).sortBy(_.app.id)` could also work and be cheaper.